RickAndMorty/Extensions/String+Ext.swift

```diff
@@ -18,6 +18,14 @@ extension String {
         return nil
     }
     
+    var pageNumber: Int? {
```

Comment:
No longer needed, check the new Property Wrapper Approach

Reply:
Ready, waiting for review.